pandas/_libs/reduction.pyx

```diff
@@ -1,9 +1,24 @@
 # -*- coding: utf-8 -*-
```

Comment:
you don't like reduce?

Reply:
I try to avoid overlap with built-in names.